entrypoint.d/50-config-generate

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ "${CONFIG_BUILD}" = true ]; then
```

Comment:
Hahaha interestingly this same recommendation was made and then reversed in the last PR. In the instance of config generation during entrypoint, we actually want to wipe any pre-existing configuration that may exist.

That said & in hindsight - I don't know of a way to change environment variables without destroying the container. A new container wouldn't have the configuration on it because it'd be a fresh image, so I think this means that we can actually use this method safely?